mindjunkies/courses/migrations/0007_remove_course_progression_remove_module_progression_and_more.py

```
@@ -0,0 +1,27 @@
+        migrations.RemoveField(
+            model_name='course',
+            name='progression',
+        ),
+        migrations.RemoveField(
+            model_name='module',
+            name='progression',
+        ),
+        migrations.AddField(
+            model_name='enrollment',
+            name='progression',
+            field=models.PositiveIntegerField(default=0, validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)]),
```

Ruff (0.8.2)

Line too long (162 > 119)

(E501)